incubator/vault/templates/serviceaccount.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: {{ template "vault.serviceAccountName" . }}
   labels:
     app: {{ template "vault.name" . }}
-    chart: {{ template "vault.chart" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
```

Comment:
This does not capture the full scope of what the `chart` function does. Would you mind adding the standard `chart` function to the `_helpers.tpl` file instead. That function reads:

```
{{/*
Create chart name and version as used by the chart label.
*/}}
{{- define "wtf.chart" -}}
{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
{{- end -}}
```

Reply:
Good call, I can do that. I'll also adjust the other labels that currently reference `{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}` to use the `vault.chart` function.